cicecore/drivers/nuopc/cmeps/ice_comp_nuopc.F90

```diff
@@ -26,6 +26,7 @@ module ice_comp_nuopc
   use ice_calendar       , only : idate, mday, time, month, daycal, time2sec, year_init
   use ice_calendar       , only : sec, dt, calendar, calendar_type, nextsw_cday, istep
   use ice_kinds_mod      , only : dbl_kind, int_kind, char_len, char_len_long
+  use ice_scam           , only : scmlat, scmlon, scol_mask, scol_frac, scol_ni, scol_nj, scol_valid, single_column
```

Comment:
Does this need to be wrapped in a CESMCOUPLED ifdef for EMC?

Reply:
done.